### PR TITLE
update french translation in strings.txt categories.txt categories_cuisines.txt 

### PR DESCRIPTION
**Pull request by Kazing**
Reviewed and updated french translation, I will continue with data/strings/types_strings.txt and do another PR later

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -2714,7 +2714,8 @@
...
     be = Вітаю!\n\nУ далучаным файле мае закладкі з дататка Organic Maps. Калі ласка, адчыніце іх калі ў вас усталяваны Organic Maps. Калі не, спампуйце дадатак для вашай прылады iOS альбо Andriod па спасылцы: https://omaps.app/get?kmz\n\nПрыемных падарожжаў з Organic Maps!
-    nl = Hallo!\n\nBijgesloten zijn mijn bladwijzers uit de Organic Maps app. Open ze als je Organic Maps geïnstalleerd hebt. Of, als je dat niet hebt, download de app voor je iOS- of Android-apparaat door deze link te volgen: https://omaps.app/get?kmz\n\nGeniet van het reizen met Organic Maps!
+    fr = Bonjour !\n\nVous trouverez ci-joint mes signets de l'appli Organic Maps. Veuillez les ouvrir si vous avez installé Organic Maps. Si vous ne l'avez pas, téléchargez l'application pour votre appareil iOS ou Android en suivant ce lien : https://omaps.app/get?kmz\n\nBon voyage avec Organic Maps !
+	nl = Hallo!\n\nBijgesloten zijn mijn bladwijzers uit de Organic Maps app. Open ze als je Organic Maps geïnstalleerd hebt. Of, als je dat niet hebt, download de app voor je iOS- of Android-apparaat door deze link te volgen: https://omaps.app/get?kmz\n\nGeniet van het reizen met Organic Maps!
```
Comment (biodranik) [inline data/strings/strings.txt:2718]: This line shifted a bit.

File: data/strings/strings.txt [hunk 2]
```diff
@@ -21892,6 +21900,7 @@
     en = Download the world map
     be = Спампаваць мапу света
     bg = Изтегляне на картата на света
+	fr = Télécharcher la carte du monde
```
Comment (biodranik) [inline data/strings/strings.txt:21903]: Can you please align it?

File: data/strings/strings.txt [hunk 3]
```diff
@@ -21902,6 +21911,7 @@
     en = Connection failure
     be = Памылка падлучэння
     bg = Неуспешно свързване
+	fr = Erreur de connexion
```
Comment (biodranik) [inline data/strings/strings.txt:21914]: Can you please align it?

File: data/strings/strings.txt [hunk 4]
```diff
@@ -21912,6 +21922,7 @@
     en = Disconnect USB cable
     be = Адлучыце USB кабель
     bg = Изключване на USB кабел
+	fr = Déconnectez le câble USB
```
Comment (biodranik) [inline data/strings/strings.txt:21925]: Can you please align it?

File: data/strings/strings.txt [hunk 5]
```diff
@@ -22160,6 +22171,7 @@
     comment = Delete track button on track edit screen
     tags = ios
     en = Delete Track
+	fr = Supprimer la route
```
Comment (biodranik) [inline data/strings/strings.txt:22174]: Can you please align it?

File: data/strings/strings.txt [hunk 6]
```diff
@@ -22169,6 +22181,7 @@
     tags = ios
     en = Track Name
     be = Назва сцежкі
+	fr = Nom de la route
```
Comment (biodranik) [inline data/strings/strings.txt:22184]: Can you please align it?

File: data/strings/strings.txt [hunk 7]
```diff
@@ -22177,6 +22190,7 @@
     tags = ios
     en = Move
     be = Перамясціць
+	fr = Déplacer
```
Comment (biodranik) [inline data/strings/strings.txt:22193]: Can you please align it?

**Comment by Kazing** [comment]
I corrected the alignement problems

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**